---
float64 x
float64 width
float64 time_interval
